_posts/2020-01-29-#17487.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: pr
 date: 2020-01-29
-title: "Coins: allow Flush() without cache drop"
+title: "allow Flush() without cache drop"
```

Comment:
This title is pretty ambiguous without stating that it's about the coins db.

Reply:
true, I'll add those two back